conans/client/tools/system_pm.py

```diff
@@ -175,6 +192,23 @@ def installed(self, package_name):
         exit_code = self._runner("dpkg-query -W -f='${Status}' %s | grep -q \"ok installed\"" % package_name, None)
         return exit_code == 0
 
+    def _platforms(self):
+        return
+
+    def _parse_package(self, package, arch, platforms):
```

Comment:
The name of the function is not helping to understand the code at all ;)

Reply:
renamed to "get_package_name"